Codeinwp/otter-blocks
# Fix dynamic CSS and script loading for FSE

Closes #652.


### Summary


Add templates to our pipeline.

### Screenshots 

----

### Test instructions


1. Insert all the blocks.
2. Customize it.
3. Check the CSS in frontend.

#### Query
```javascript
new QueryQA().select('blocks').run()
```

---- 

### Checklist before the final review
